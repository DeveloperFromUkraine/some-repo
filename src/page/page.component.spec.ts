import { ComponentFixture, TestBed } from '@angular/core/testing';
import { PageComponent } from '../index';
import { ComponentTest } from '../../test/test-bed/component';

describe('Page', () => {
  let fixture: ComponentFixture<PageComponent>;

  beforeEach(async () => {
    await ComponentTest.createTestBed([], [PageComponent]);
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(PageComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
