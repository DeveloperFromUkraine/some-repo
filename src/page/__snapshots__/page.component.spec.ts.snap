// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageComponent should match snapshot 1`] = `
<ign-page>
  
      
    
</ign-page>
`;
